### PR TITLE
feat: events page

### PR DESCRIPTION
**Pull request by nerucheva**
This PR adds the Events page without integration with Google calendar
[Preview](https://deploy-preview-397--ebpfio.netlify.app/events/)

### CODE REVIEW

**Comment by xmulligan** [comment]
Great, can you also add the event branding for each card? I would keep it basic so the logo or image of the event.

For the drop down at the top, can you limit it to just overview and then each high level event rather than listing each year? "Overview, eBPF Summit, Cloud Native eBPF Day, eBPF Track (LPC), bpfconf, and IETF"

**Comment by xmulligan** [comment]
Also can you make the newest events the featured ones at the top?

File: src/components/pages/events/hero/hero.jsx [hunk 1]
```diff
@@ -1,11 +1,11 @@
 import PropTypes from 'prop-types';
-import React from 'react';
+import React, { useMemo } from 'react';
 
 import ListCard from './list-card';
 import MainCard from './main-card';
 
 const Hero = ({ items }) => {
-  const mainItem = items.shift();
+  const mainItem = useMemo(() => items.shift(), [items]);
```
Comment (nerucheva) [inline src/components/pages/events/hero/hero.jsx:8]: Do we really need to memoize the mainItem value?

File: src/components/shared/header/header.jsx [hunk 2]
```diff
@@ -73,7 +73,7 @@ const Header = ({ items, isMobileMenuOpen, onBurgerClick, fullWidthBottomBorder
                             {items.map(({ text, to, target }) => (
                               <li key={text}>
                                 <Link
-                                  className="flex py-2.5 px-5 text-[15px] leading-none lg:text-sm"
+                                  className="blabla flex py-2.5 px-5 text-[15px] leading-none lg:text-sm"
```
Comment (nerucheva) [inline src/components/shared/header/header.jsx:76]: Can we remove this `blabla` class?

**Review by nikvoblikov — APPROVED** [review]

**Comment by xmulligan** [comment]
closing in favor of #403

**Pull request closed without merging**